consensus/hotstuff/votecollector/staking_vote_processor.go

```diff
@@ -29,14 +29,14 @@ import (
 // by `votecollector.VoteProcessorFactory` which adds the logic to verify
 // the proposer's vote (decorator pattern).
 type stakingVoteProcessorFactoryBase struct {
-	committee   hotstuff.Committee
+	committee   hotstuff.Replicas
```

Comment:
same argument as for [combined_vote_processor_v2.go](https://github.com/onflow/flow-go/blob/4de4231e43a65fe672a54aab4f84971ecac03db6/consensus/hotstuff/votecollector/combined_vote_processor_v2.go), I think we should use the `DynamicCommittee` here to reject votes from ejected replicas. 

⚠️ not to forget updating https://github.com/onflow/flow-go/blob/4de4231e43a65fe672a54aab4f84971ecac03db6/consensus/hotstuff/votecollector/staking_vote_processor.go#L58